downstream/modules/automation-hub/proc-obtain-images.adoc

```diff
@@ -0,0 +1,46 @@
+
+
+[id="obtain-images"]
+
+
+= Obtaining images for use in {HubName}
+
+
+[role="_abstract"]
+Before you can push container images to your private {HubName}, you must first pull them from an existing registry and tag them for use. This example details how to pull an image from the Red Hat Ecosystem Catalog (registry.redhat.io).
```

Comment:
Do we put backticks around `registry.redhat.io` ?  OpenShift docs use them.

Reply:
But the Container Registry Authentication docs don't add backticks to inline mentions of registry.redhat.io